src/js/pensions/helpers.jsx

```diff
@@ -61,7 +61,7 @@ export function fileHelp({ formContext }) {
   return (
     <p>
       Files we accept: pdf, jpg, png<br/>
-      Maximum file size: 2MB
+      Maximum file size: 20MB
```

Comment:
I didn't do that mainly because there's not an obvious way to tie the two together. This is a regular description field and doesn't know anything about the file field. On the burial form, this description covers two file fields, so we can't really have the common file definition include this description, either.

We could make a higher order component that you pass in file types and a max size, but that's only slightly (if at all) better.